linera-chain/src/manager.rs

```diff
@@ -125,13 +125,13 @@ pub struct ChainManager {
     pub timeout: Option<TimeoutCertificate>,
     /// Latest vote we have cast, to validate or confirm.
     #[debug(skip_if = Option::is_none)]
-    pub pending: Option<Vote>,
+    pub pending: Option<Vote<CertificateValue>>,
```

Comment:
This will be tricky to get rid of as we set all three types of `CertificateValue` here.

Reply:
The `timeout_vote` is already separate. We could split up the other two as well.

Reply:
Agreed to do in a seperate PR